### PR TITLE
[APP-733] Restrict validation rules for Hashes

### PR DESCRIPTION
**Pull request by psyxoz**
Related to https://peopleforce.atlassian.net/browse/APP-733

### CODE REVIEW

File: spec/validate_params/validatable/nested_spec.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
 require "fixtures/controllers/nested_controller"
 require "fixtures/controllers/nested_override_controller"
 
-RSpec.describe ValidateParams::Validatable do
+xdescribe ValidateParams::Validatable do
```
Comment (acetinick) [inline spec/validate_params/validatable/nested_spec.rb:6]: Fix

**Review by acetinick — CHANGES_REQUESTED** [review]

File: spec/validate_params/validatable/nested_spec.rb (hunk 1 above)
Comment (psyxoz) [inline spec/validate_params/validatable/nested_spec.rb:6]:
```suggestion
describe ValidateParams::Validatable do
```

Comment (psyxoz) [reply spec/validate_params/validatable/nested_spec.rb]: Done ✅

File: spec/validate_params/validatable/nested_spec.rb [hunk 2]
```diff
@@ -3,7 +3,7 @@
 require "fixtures/controllers/nested_controller"
 require "fixtures/controllers/nested_override_controller"
 
-RSpec.describe ValidateParams::Validatable do
+describe ValidateParams::Validatable do
```
Comment (psyxoz) [inline spec/validate_params/validatable/nested_spec.rb:6]:
```suggestion
xdescribe ValidateParams::Validatable do
```

**Pull request merged**